.github/workflows/compile-mermaid.yml

```diff
@@ -42,6 +42,8 @@ jobs:
     - name: Convert all svg files to png before uploading for automatic inspection
       run: |
         npm install convert-svg-to-png
+        # convert-svg-to-png tries to create files by changing their extension, which could clash with already-existing PNG files
+        for png_file in "${{env.INPUT_DATA}}"/*.png; do echo "Moving $png_file..."; mv "$png_file" "$png_file.bak"; done
         $(npm bin)/convert-svg-to-png ${{env.INPUT_DATA}}/*.svg
```

Comment:
Please, use the file name instead of `${{env.INPUT_DATA}}/*.svg,` e.g. `${{env.INPUT_DATA}}/<my-filename>.svg`
Repeat this statement `$(npm bin)/convert-svg-to-png ${{env.INPUT_DATA}}/<filename>.svg` as many times, as you have different input files. Maybe you can do something like this `$(npm bin)/convert-svg-to-png ${{env.INPUT_DATA}}/<common-part-of-filename>*.svg` if there are multiple files to be converted.

Reply:
Done